File: JAVASCRIPT/Questões uri/1985.js
const lenha = (line) => line.split(" ").map(a => +a)
let produtos=+lines.shift()
let conta=0
for(let cont=0; cont<produtos;cont++){
    let [cod,qtd] = lenha(lines.shift())
    if(cod==1001){
        conta=conta+(qtd*1.50)
    }
    if(cod==1002){
        conta=conta+(qtd*2.50)
    }
    if(cod==1003){
        conta=conta+(qtd*3.50)
    }
    if(cod==1004){
        conta=conta+(qtd*4.50)
    }
    if(cod==1005){
        conta=conta+(qtd*5.50)
    }
}
console.log(conta.toFixed(2))

File: ESTUDO DE LINGUAGENS/PYTHON/Questões Uri/1985.py
qtd=int(input())
conta=float(0)
for cont in range(0,qtd,1):
  cod,quan=map(int, input().split(" "))
  if cod==1001:
    conta=conta+(quan*1.50)
  if cod==1002:
    conta=conta+(quan*2.50)
  if cod==1003:
    conta=conta+(quan*3.50)
  if cod==1004:
    conta=conta+(quan*4.50)
  if cod==1005:
    conta=conta+(quan*5.50)
print("%0.2f"%conta)